app/assets/javascripts/jquery_overrides.js

```diff
@@ -1,6 +1,10 @@
 jQuery.evalLog = function (text) {
   try {
-    jQuery.globalEval(text.slice('throw "error";'.length));
+    if (text.match('^throw "error";')) {
```

Comment:
Yeah I think we may have to add some check that it's really there before we slice.. But honestly.. I'm not sure about the security implications of letting this be completely optional.. @martinpovolny , @isimluk  maybe you know more?

If there's none, cool, if some, maybe adding that throw to that proxy page does make sense..